celery/backends/couchdb.py

```diff
@@ -91,13 +93,16 @@ def get(self, key):
             return None
 
     def set(self, key, value):
-        data = {'_id': key, 'value': value}
+        # data = {'_id': key, 'value': value}
+        data = json.loads(value)
```

Comment:
I'm not familiar with the anatomy of a results backend class, so I'm not at all surprised if there's a better place to put that code.

The result of this code (wherever it goes) should be necessary, however, unless the intention is that the value of 'value' is supposed to be serialized within the serialized json string.  Like this:

```
{
  "_id": "celery-task-meta-2c040203-a07a-4a92-9fa1-3269839e2c54",
  "_rev": "1-6b9803d5b1a4ed51a5274ef7c567a332",
  "value": "{'status': 'SUCCESS', 'traceback': null, 'result': {}, 'task_id': '2c040203-a07a-4a92-9fa1-3269839e2c54', 'children': []}"
}
```

If that's not the intention, my proposed change would instead result in the CouchDB document looking like this:

```
{
   "_id": "celery-task-meta-2c040203-a07a-4a92-9fa1-3269839e2c54",
   "_rev": "1-6b9803d5b1a4ed51a5274ef7c567a332",
   "status": "SUCCESS",
   "result": { },
   "task_id": "2c040203-a07a-4a92-9fa1-3269839e2c54",
   "traceback": null,
   "children": [
   ]
}
```